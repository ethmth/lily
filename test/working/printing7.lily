let a: float = print(5.6)

print(a)