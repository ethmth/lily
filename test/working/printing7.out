5.600000
5.600000